packages/react-button/src/components/MenuButton/useExpanded.ts

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { getCode, ArrowDownKey } from '@fluentui/keyboard-key';
+import { useControllableValue, useMergedRefs } from '@fluentui/react-utilities';
+import { MenuButtonState } from './MenuButton.types';
+
+export type ExpandedState = {
```

Comment:
`MenuTrigger` should remove the need to have this state hook

Reply:
Removed this hook.